src/qcodes_contrib_drivers/drivers/HP/HP_4194A.py

```diff
@@ -0,0 +1,684 @@
+import warnings
+from typing import Any, Union, Optional, Callable
+from typing_extensions import Literal
+
+# from qcodes import ParameterWithSetpoints, VisaInstrument
+from qcodes.instrument import ParameterWithSetpoints, VisaInstrument
+from qcodes.instrument.parameter import Parameter  # , invert_val_mapping
+from qcodes.utils.helpers import create_on_off_val_mapping
+from qcodes.utils.validators import Enum, Ints, Numbers, Arrays  # , Lists,
```

Comment:
Similar to your other pr could you update these prs to fiit the official definitions from qcodes